language-server/modules/langserver-core/src/test/java/org/ballerinalang/langserver/codeaction/AbstractCodeActionTest.java

```diff
@@ -99,34 +106,46 @@ public void test(String config, String source) throws IOException, WorkspaceDocu
 
                 // Match title
                 String actualTitle = right.get("title").getAsString();
-                if (!expTitle.equals(actualTitle)) {
+                if (!expected.title.equals(actualTitle)) {
                     continue;
                 }
+
+                // We have to make sure the title is a match since we are checking against all the
+                // code actions received.
+                actual.title = actualTitle;
```

Comment:
This statement will not be reached when the titles are not matching in line 109. So the actual object would not have a title in that case. I think, this would be a problem we call updateConfig() in line 183.

Reply:
Fixed the logic.